doc/api/prev_api_changes/api_changes_3.1.0.rst

```diff
@@ -863,7 +863,7 @@ Use the standard library's docstring manipulation tools instead, such as
 These are considered internal and will be removed from the public API in a
 future version.
 
-- `.projections.process_projection_requirements`
+- ``projections.process_projection_requirements``
```

Comment:
these should technically be in a separate PR but I'm not going to be nitpicky with that.

Reply:
If one wants to be very strict yes, but it's just changing link to literal, which can be done anytime is not strictly related to the deprecation of that function. I saved the hassle of cherry-pickling parts of the file. :smile: